### ISSUE TITLE
getTransportDetails - Invalid value for enum

### ISSUE THREAD

**Issue by HeBuguM** [jlevers/selling-partner-api#126]
Another discrepancy in the response/documentation breaks the request.

**Request:** 
SellingPartnerApi\Api\FbaInboundApi->getTransportDetails('ShipmentId');

**Error:** 
Invalid value for enum '\SellingPartnerApi\Model\FbaInbound\SellerFreightClass', must be one of: '50', '55', '60', '65', '70', '77.5', '85', '92.5', '100', '110', '125', '150', '175', '200', '250', '300', '400', '500'

**Response Body:** 
```
{
  "payload": {
    "TransportContent": {
      "TransportHeader": {
        "SellerId": "...",
        "ShipmentId": "...",
        "IsPartnered": true,
        "ShipmentType": "LTL"
      },
      "TransportDetails": {
        "PartneredLtlData": {
          "Contact": {
            "Name": "...",
            "Phone": "....",
            "Email": "...."
          },
          "BoxCount": 30,
          "FreightReadyDate": "Tue Nov 09 00:00:00 GMT 2021",
          "PalletList": [
            {
              "Dimensions": {
                "Length": 48.0,
                "Width": 40.0,
                "Height": 56.0,
                "Unit": "IN"
              },
              "Weight": {
                "Value": 960.0,
                "Unit": "pounds"
              }
            }
          ],
          "TotalWeight": {
            "Value": 960.0,
            "Unit": "pounds"
          },
          "SellerDeclaredValue": {
            "CurrencyCode": "USD",
            "Value": 5000.0
          },
          "PreviewPickupDate": "Wed Nov 10 00:00:00 GMT 2021",
          "PreviewFreightClass": "85.0",
          "AmazonReferenceId": "15625378241",
          "IsBillOfLadingAvailable": true,
          "PartneredEstimate": {
            "Amount": {
              "CurrencyCode": "USD",
              "Value": 97.09
            },
            "VoidDeadline": "2021-11-09T16:03:46Z"
          }
        }
      },
      "TransportResult": {
        "TransportStatus": "CONFIRMED"
      }
    }
  }
}
```

**PreviewFreightClass** value is '**85.0**' instead of '**85**' 

Defining constants like `const _85_0 = '85.0';` for all possible values in **\SellingPartnerApi\Model\FbaInbound\SellerFreightClass** is kind of silly but... Probably you can find a better solution?

**Comment by jlevers** [comment]
Should be fixed in `v4.1.7`. Please give it a try and close the issue if it's resolved. Thanks :)

**Comment by HeBuguM** [comment]
All good thank you!
